resources/datadog-integrations/msi/source.wxs.erb

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <!-- This is how we include wxi files -->
+  <?include "parameters.wxi" ?>
+
+  <!--
+    Id="*" is to enable upgrading. * means that the product ID will be autogenerated on each build.
+    Name is made of localized product name and version number.
+  -->
+  <Product Id="*"
+           Name="$(var.PackageName)"
+           Language="1033"
+           Version="$(var.VersionNumber)"
+           Manufacturer="!(loc.ManufacturerName)"
+           UpgradeCode="$(var.UpgradeCode)"
+           Codepage="1252">
+
+    <!--
+      Define the minimum supported installer version (2.0).
+      The install should be done for the whole machine not just the current user
+    -->
+    <Package Id="*"
+             Keywords="Installer"
+             Comments="Copyright 2015 Datadog, Inc."
+             Description="Datadog $(var.Description)"
+             Manufacturer="!(loc.ManufacturerName)"
+             InstallerVersion="400"
+             Languages="1033"
+             InstallPrivileges="elevated"
+             Compressed="yes" />
+
+    <!-- This removes entirely the previous version -->
+    <Upgrade Id="$(var.UpgradeCode)">
+        <UpgradeVersion OnlyDetect='no'
+                        Property='PREVIOUSFOUND'
+                        Minimum='1.0.0' IncludeMinimum='yes'
+                        Maximum="$(var.DisplayVersionNumber)" IncludeMaximum='no'/>
+    </Upgrade>
+
+    <!-- No need to restart Windows after the installation -->
+    <Property Id="MSIRESTARTMANAGERCONTROL" Value="Disable" />
+
+    <!-- Make sure that we get rid of any older installation of the Agent -->
+    <MajorUpgrade
+      DowngradeErrorMessage="A later version of [ProductName] is already installed. Setup will now exit." />
+
+    <Media Id="1" Cabinet="agent.cab" EmbedCab="yes" />
+
+    <Directory Id="TARGETDIR" Name="SourceDir">
+      <Directory Id="ProgramFiles64Folder">
+        <Directory Id="APPLICATIONROOTDIRECTORY" Name="Datadog">
+          <Directory Id="INTEGRATIONSROOT" Name="Integrations">
+            <Directory Id="PROJECTLOCATION" Name="$(var.IntegrationName)"> 
+            </Directory>
+          </Directory>
+        </Directory>
+      </Directory>
+      <Directory Id="CommonAppDataFolder" SourceName="CommonAppData">
+        <Directory Id="APPLICATIONDATADIRECTORY" Name="Datadog"/>
+      </Directory>
+    </Directory>
+
+
+    <!-- Delete application when uninstalling -->
+    <!--
+      RemoveFolderEx requires that we "remember" the path for uninstall.
+      Read the path value and set the APPLICATIONROOTDIRECTORY property with the value.
+    -->
+    <Property Id="APPLICATIONROOTDIRECTORY">
```

Comment:
We need it for the same reason we need it in the core agent, in case there are files created at runtime (like .pyc).  But, it wasn't functioning properly.  Redid that section so that now the directory is blown away on uninstall, .pycs and all.